pkg/client/cache/listers.go

```diff
@@ -225,6 +225,59 @@ func (s *StoreToReplicationControllerLister) GetPodControllers(pod *api.Pod) (co
 	return
 }
 
+// StoreToDaemonLister gives a store List and Exists methods. The store must contain only Daemons.
+type StoreToDaemonLister struct {
+	Store
+}
+
+// Exists checks if the given dc exists in the store.
```

Comment:
replace "dc" by daemon controller. I have seen dc being used for deploymentController as well :)